### ISSUE TITLE
ne_10m_admin_0_countries_points

### ISSUE THREAD

**Issue by teceP** [geosolutions-it/osm-styles#4]
Hey, anyone knows, where I could get this shapefile from?

"ne_10m_admin_0_countries_points"

I found [this](https://github.com/klokantech/vector-tiles-sample/tree/master/countries.tm2source/files) repo, but it didnt worked for me.

**Comment by estemka** [comment]
https://www.naturalearthdata.com/downloads/10m-cultural-vectors/10m-admin-0-countries/ 
ne = natural earth

**Comment by teceP** [comment]
Yeah, I tried this already.

When I try to make a layer preview with this shapefile and the css-file "states", I get following error:

> The requested Style can not be used with this layer. The style specifies and attribute of area and the layer is: ohdm:ne_10m_admin_0_countries

**Comment by astropcrb** [comment]
I am also getting this issue. Looks like this shp file is not available anymore.

**Comment by aaime** [comment]
Wondering... scrolling at the bottom of:
https://www.naturalearthdata.com/downloads/10m-cultural-vectors/

there is this one, which seems close enough:
https://www.naturalearthdata.com/http//www.naturalearthdata.com/download/10m/cultural/ne_10m_admin_0_label_points.zip

Maybe give it a try? By the name/description seems to be same/similar content.

**Comment by astropcrb** [comment]
Thanks. I used the one you recommended. 

Not quite a 1:1 translation. Here is the modified CSS I created. The new Shapefile didn't have the handy "area" attribute.

`@mode 'Flat';

[@sd < 120M][@sd > 150k]{
  label: [strToUpperCase(sr_br_name)];
  label-anchor: 0.5 0.5;
  label-group: true;
  font-fill: #777777;
  font-weight: bold;
  font-size: [interpolate(env('wms_scale_denominator', 100000000), 800000, 14, 100000000, 12)];
  font-family: 'Noto Sans';
  halo-color: #ffffff;
  halo-radius: 1.5;
  halo-opacity: 0.8;
  -gt-label-padding: 7;
  -gt-label-auto-wrap: 90;
  label-priority: [scalerank];
  sort-by: scalerank;
}
`
